src/types.ts

```diff
@@ -1,4 +1,64 @@
-export interface Translator {
-  get: (str: string, args?: string[]) => string;
-  language: () => string;
+import React from 'react';
+import { stardust } from '@nebula.js/stardust';
+
+export interface TableCell {
+  qText: string | undefined;
+  qAttrExps: EngineAPI.INxAttributeExpressionValues;
+  qElemNumber: number;
+  rowIdx: number;
+  colIdx: number;
+  isSelectable: boolean;
+  rawRowIdx: number;
+  rawColIdx: number;
+  prevQElemNumber: number;
+  nextQElemNumber: number;
+}
+
+export interface ExtendedSelectionAPI extends stardust.ObjectSelections {
+  on(eventType: string, callback: () => void): void;
+  removeListener(eventType: string, callback: () => void): void;
+}
+
+export interface SelectedRows {
+  [key: string]: number;
+}
```

Comment:
This one is a bit replaceable. You can use the built-in type `Record<string, number>` instead.

Reply:
I knew I had seen something like that, will update